integration-tests/esbuild/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "esbuild-dd-trace-demo",
+  "private": true,
+  "version": "1.0.0",
+  "description": "basic example app bundling dd-trace via esbuild",
+  "main": "app.js",
+  "scripts": {
+    "build": "DD_TRACE_DEBUG=true node ./build.js",
+    "built": "DD_TRACE_DEBUG=true node ./out.js",
+    "raw": "DD_TRACE_DEBUG=true node ./app.js",
+    "link": "pushd ../.. && yarn link && popd && yarn link dd-trace",
+    "request": "curl http://localhost:3000 | jq"
+  },
+  "keywords": [
+    "esbuild",
+    "apm"
+  ],
+  "author": "Thomas Hunter II <tlhunter@datadog.com>",
+  "license": "ISC",
+  "dependencies": {
+    "esbuild": "0.16.12",
+    "express": "^4.16.2"
```

Comment:
These two entries are required for the linter to pass, even though they're never installed. Instead, the dev dep packages available in the root of the repo are used.

I could simply delete the package.json file but I kind of like the scripts section as it provides docs of sorts.

Reply:
I agree, and also it would be good in any case at some point to decouple this from the parent dependencies.